networking/util/net.go

```diff
@@ -22,12 +22,13 @@ import (
 
 type Net struct {
 	Filename string
-	Name     string `json:"name,omitempty"`
-	Type     string `json:"type,omitempty"`
+	Name     string   `json:"name,omitempty"`
+	Type     string   `json:"type,omitempty"`
 	IPAlloc  struct {
 		Type   string `json:"type,omitempty"`
 		Subnet string `json:"subnet,omitempty"`
-	}               `json:"ipAlloc"`
+	}                 `json:"ipAlloc,omitempty"`
+	Routes   []string `json:"routes,omitempty"`
 }
 
 func LoadNet(path string, n interface{}) error {
```

Comment:
Because the json config file may have other keys (plugin specific) besides the ones listed here. So the expected usage for that case is to embed this struct in plugin specific struct but still be able to call LoadNet.